pl-csb-data/logger.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import typing as tp
+
+
+map_level = {
+    "DEBUG": logging.DEBUG,
+    "INFO": logging.INFO,
+    "WARNING": logging.WARNING,
+    "ERROR": logging.ERROR,
+    "CRITICAL": logging.CRITICAL
+}
+
+DEFAULT_LOGGER_LEVEL = logging.INFO
+
+
+def set_up_logger(name: str, logger_level: tp.Optional[str] = None) -> logging.Logger:
+    level = map_level.get(logger_level, DEFAULT_LOGGER_LEVEL) if logger_level else DEFAULT_LOGGER_LEVEL
+
+    logger = logging.getLogger(name)
+    logger.setLevel(level)
+    
+    formatter = logging.Formatter('%(asctime)s [%(process)s] [%(levelname)s] - %(message)s')
```

Comment:
```suggestion
    formatter = logging.Formatter("%(asctime)s [%(process)s] [%(levelname)s] - %(message)s")
```
PEP 8 doesn’t prefer single quotes over double quotes, so this change is for consistency.